datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py

```diff
@@ -152,7 +152,15 @@ def create(ctx, name, integration_type, location, manifest_v2, non_interactive,
         else:
             # Fill in all common non Marketplace fields
             template_fields['pricing_plan'] = ''
-            template_fields['author_info'] = ''
+            template_fields[
+                'author_info'
+            ] = """
+  "author": {
+    "support_email": "help@datadoghq.com",
```

Comment:
Is there a way to distinguish between extras and core integrations? Sometimes extras maintainers don't update these fields ([V1 example](https://github.com/DataDog/integrations-extras/blob/master/zscaler/manifest.json#L3)).